openmc/region.py

```diff
@@ -46,6 +46,25 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
+    def get_surfaces_from_region(self, surfaces = OrderedDict()):
```

Comment:
Yes, PEP8 says no whitespace around = in an argument list. Also, you should not provide a [mutable default value to an argument](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html).